Makefile

```diff
@@ -1,8 +1,30 @@
 # Use bash as shell
 SHELL = /bin/bash
 
+# Check for dirty :D 
+define check_dirty
+	GIT_SHA=$$(git rev-parse HEAD) || { \
+		GIT_HASH=$${GIT_SHA:-NO_SHA}; \
+	}; \
+	if [ -z "$$GIT_HASH" ]; then \
+		GIT_DIRTY=$$(git diff --stat); \
+		if [ -n "$$GIT_DIRTY" ]; then \
+			DIRTY="true"; \
+		else \
+			DIRTY="false"; \
+		fi; \
+	fi; \
+	echo $$DIRTY
+endef
+
 # Authorino version
-VERSION = $(shell git rev-parse HEAD)
+AUTHORINO_VERSION = v0.17.3-dev # change as version increments
```

Comment:
I would keep versioning at golang level

wdyt @guicassolato ?

Reply:
I'll hold on implementing here until Gui gets back to us :)

Reply:
I'm OK with "X.Y.Z" version, i.e. without leading "v".

Be careful with other other possible uses of this variable, like in the tagging of the Docker image, which must continue to follow the pattern `vX.Y.Z`.

I'm not sure about hard coding the version inside the Makefile. We didn't need that before. It was either a SHA for latest or an actual semantic version number (from the name of the branch/tag.)

Also, no need to rename the variable `AUTHORINO_VERSION` IMO. If it's inside Authorino's Makefile, `VERSION` must refer to the Authorino version.

Reply:
@guicassolato all good suggestions, and thank you for the insight on the docker tagging. 

> I'm OK with "X.Y.Z" version, i.e. without leading "v".

OK, makes sense, for semantics. I'll move the v down to the string that displays the version.

> I'm not sure about hard coding the version inside the Makefile. We didn't need that before. It was either a SHA for latest or an actual semantic version number (from the name of the branch/tag.)

OK sure, I'll move `VERSION` back to global level and change to latest.